pom.xml

```diff
@@ -51,7 +51,13 @@
     <dependency>
       <groupId>com.squareup.retrofit</groupId>
       <artifactId>retrofit</artifactId>
-      <version>1.6.1</version>
+      <version>2.0.0-beta1</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
```

Comment:
You are missing the Retrofit Gson converter artifact so this isn't going to work